klog.go

```diff
@@ -874,8 +881,20 @@ func (sb *syncBuffer) Sync() error {
 	return sb.file.Sync()
 }
 
+//IsFileReachedMaxSize checks whether the file exceeds the max size limitation.
```

Comment:
nit: space after comment
```suggestion
// IsFileReachedMaxSize checks whether the file exceeds the max size limitation.
```